imu:
    topic: "/imu_data"
    time_offset: 0.0
    update_rate: 10.0
    T_imu_to_base:
        [
            [1.0, 0.0, 0.0, 0.100215],
            [0.0, 1.0, 0.0, 0.0056],
            [0.0, 0.0, 1.0, 0.1666],
            [0.0, 0.0, 0.0, 1.0],
        ]

    # 加速度 轴偏 噪声标准差 偏置标准差 静态偏置
    acc_sm: [[1.0, 0.0, 0.0], [0.0, 1.0, 0.0], [0.0, 0.0, 1.0]]
    sigma_acc_noise: 0.01
    sigma_acc_bias: 0.01
    acc_bias: [0.0, 0.0, 0.0]

    # 角速度 轴偏 噪声标准差 偏置标准差 静态偏置
    gyr_sm: [[1.0, 0.0, 0.0], [0.0, 1.0, 0.0], [0.0, 0.0, 1.0]]
    sigma_gyr_noise: 0.001
    sigma_gyr_bias: 0.001
    gyr_bias: [0.0, 0.0, 0.0]
